bing-speech/speech.1.0.0.d.ts

```diff
@@ -0,0 +1,1384 @@
+	
+	/**
+	 * 
+	 */
+	interface HandleJSONWebResponse0 {
+				
+		/**
+		 * 
+		 */
+		responseType : string;
+				
+		/**
+		 * 
+		 */
+		onload(): void;
+	}
+}
+declare namespace Bing{
+	// Bing.LuisClient.!0
+	
+	/**
+	 * 
+	 */
+	interface LuisClient0 {
+				
+		/**
+		 * 
+		 */
+		serviceUri : string;
+				
+		/**
+		 * 
+		 */
+		clientVersion : string;
+				
+		/**
+		 * 
+		 */
+		authenticationScheme : string;
+	}
+}
+declare namespace Bing.HttpClient.prototype{
+	// Bing.HttpClient.prototype.dispatchResult.!0
+	
+	/**
+	 * 
+	 */
+	interface DispatchResult0 {
+				
+		/**
+		 * 
+		 */
+		resultIndex : number;
+		
+		/**
+		 * 
+		 */
+		results : {
+						
+			/**
+			 * 
+			 */
+			length : number;
+		}
+				
+		/**
+		 * 
+		 */
+		status : number;
+	}
+}
+declare namespace Bing.NaclClient.prototype{
+	// Bing.NaclClient.prototype.postMessage.!0
+	type PostMessage0 = Array<string>;
+}
+declare namespace Bing{
+	// Bing.createSpeech.!ret
+	
+	/**
+	 * 
+	 */
+	interface CreateSpeechRet {
+				
+		/**
+		 * 
+		 */
+		preferences : /* Bing.NaclClient.preferences */ any;
+				
+		/**
+		 * 
+		 */
+		onintent : /* Bing.NaclClient.onintent */ any;
+	}
+}
+declare namespace Bing.Speech{
+	// Bing.Speech.onresult.!0
+	
+	/**
+	 * 
+	 */
+	interface Onresult0 {
+				
+		/**
+		 * 
+		 */
+		resultIndex : number;
+		
+		/**
+		 * 
+		 */
+		results : {
+						
+			/**
+			 * 
+			 */
+			length : number;
+		}
+				
+		/**
+		 * 
+		 */
+		status : number;
+	}
+}
+declare namespace Bing.Speech.Onresult0.results{
+	// Bing.Speech.onresult.!0.results.<i>.<i>
+	
+	/**
+	 * 
+	 */
+	interface ResultsII {
+				
+		/**
+		 * 
+		 */
+		confidence : number;
+	}
+}
+declare namespace Bing.NaclClient{
+	// Bing.NaclClient.onintent.!0
+	
+	/**
+	 * 
+	 */
+	interface Onintent0 {
+	}
+}
+
+/**
+ * ----------------------------------------------------------------------
+ *  Microsoft Speech SDK
+ *  ====================
+ * 
+ * 
+ *  FEATURES
+ *  --------
+ *  * Short-form recognition.
+ *  * Long-form dictation.
+ *  * Recognition with intent.
+ *  * Integrated microphone support.
+ *  * External audio support.
+ * 
+ *  LICENSE
+ *  -------
+ *  � 2015 Microsoft. All rights reserved.
+ *  This document is provided �as-is�. Information and views expressed in this document, including URL and other Internet Web site references, may change without notice.
+ *  Some examples depicted herein are provided for illustration only and are fictitious.  No real association or connection is intended or should be inferred.
+ *  This document does not provide you with any legal rights to any intellectual property in any Microsoft product. You may copy and use this document for your internal, reference purposes. This
+ *  document is confidential and proprietary to Microsoft. It is disclosed and can be used only pursuant to a non-disclosure agreement.
+ * ----------------------------------------------------------------------
+ */
+declare var Microsoft : {
```

Comment:
namespaces can't be nested inside object types. should be `declare namespace` like the others, I think